<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDetailsPopup</class>
 <widget class="QWidget" name="OrderDetailsPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customer Details</string>
  </property>
  <widget class="QFrame" name="container_frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QFrame#container_frame {
       background-color: white;
       border: 2px solid #3855DB;
       border-radius: 20px;
     }
    </string>
   </property>
   <widget class="QFrame" name="title_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QFrame#title_bar {
        background-color: #3855DB;
        border-top-left-radius: 18px;
        border-top-right-radius: 18px;
      }
     </string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>440</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; font-size: 18px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Order Details: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QPushButton { background-color: transparent; color: white; font-size: 20px; border: none; }
       QPushButton:hover { background-color: rgba(255,255,255,0.2); border-radius: 15px; }
      </string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="print">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
            background-color: #d9d9d9; /* Example background color */
            color: #737373; /* Example text color */
            border-radius: 15px; /* Adjust this value for desired roundness */
}
QPushButton:hover {
    background-color: #9c9b9b; /* Color when the mouse hovers over it */

}
QPushButton:pressed {
    background-color: #FFFFFF; /* Color when the button is clicked down */
	color: #9c9b9b;
	border-style: outset;
	border-width: 3px;
    border-radius: 15px; /* Adjust this value for desired roundness */
    border-color: #9c9b9b;
}</string>
     </property>
     <property name="text">
      <string>Print as PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="content_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_order_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Order Info</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_status">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>41</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_payment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Payment Method: </string>
     </property>
    </widget>
    <widget class="QLabel" name="value_payment">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_order_items">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Order Items</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_total_orders">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="value_total_spent">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>175</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="value_last_order">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>250</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_summary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>275</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Summary</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_grand">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Grand Total:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_orderby">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordered By:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_process">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Processed By:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="ordertable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>155</y>
       <width>441</width>
       <height>118</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QTableWidget {
        background-color: white;
        color: black;
        gridline-color: black;
        border: 2px solid black;
    }

    QTableWidget::item {
        background-color:white;
		padding: 1px;
        color: black;
    }

    QHeaderView::section {
        background-color: white;
        color: black;
        padding: 1px;
        border: 1px solid black;
		 border-radius: 20px;
    }</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>29</number>
     </attribute>
     <column>
      <property name="text">
       <string>Service</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Details</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Amount</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_wash">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>295</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wash:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>315</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subtotal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Address to Deliver</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_personal_info_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Process Info</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="value_orderby">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="value_procesd">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="value_wash">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>295</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>₱0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="value_subtotal">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>315</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>₱0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="value_subtotal_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>360</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>₱0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="value_subtotal_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>340</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>₱0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="value_date_deliver">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_picked">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Picked up:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_datepicked">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_deliv">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Delivered:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_address_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_order_created">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_order_created">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Order Created:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images/Resc_rc.qrc"/>
 </resources>
 <connections/>
</ui>
